XamarinCommunityToolkit/Xamarin.CommunityToolkit.csproj

```diff
@@ -43,7 +47,7 @@
   <PropertyGroup Condition=" '$(Configuration)'=='Release' ">
     <!-- Manage TargetFrameworks for distribution (Release Mode) -->
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)'=='Release' And '$(OS)' == 'Windows_NT' ">
+  <PropertyGroup Condition=" '$(Configuration)'=='Release' AND '$(OS)' == 'Windows_NT' ">
```

Comment:
Is there any reason you made changes like these in the csproj file? Or did that happen automatically?

Reply:
I did add them manually i was not able to build the project on my mac without them

Reply:
@AndreiMisiukevich We made changes like this somewhere else, right? Did that not get merged?

Reply:
Unfortunately it is not merged yet

Reply:
I am fine with merging them here 👍